tools/run_tests/xds_k8s_test_driver/framework/test_cases/session_affinity_util.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2023 gRPC authors.
```

Comment:
Todo: move `GammaXdsKubernetesTestCase` to this folder (`framework/test_cases`) and make make functions in this file methods of `GammaXdsKubernetesTestCase`.

Reply:
Next PR.